src/plugins/condition/components/ConditionEdit.vue

```diff
@@ -262,18 +301,22 @@ export default {
             }
         },
         operationKeyChange(ev) {
-            if (ev.target.value !== 'isUndefined' && ev.target.value !== 'isDefined') {
```

Comment:
A function is an action and so the name should indicate that.

Reply:
Done